src/harness/testcases/WINNF_FT_S_EXZ_testcase.py

```diff
@@ -0,0 +1,443 @@
+        'registrationRequests_N3' : devices_N3,
+        'registrationRequests_N4' : devices_N4, 
+        'exclusionZoneRecords' : ex_zones,
+        'frequencyRange' : freq_ranges,
+        'grants_N2' : grants_N2,
+        'grants_N3' : grants_N3, 
+        'grants_N4' : grants_N4 
+      }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_EXZ_1_default_config)
+  def test_WINNF_FT_S_EXZ_1(self,config_filename):
+    """Injected Exclusion Zones
+
+    The response should be SUCCESS for CBSDs located outside 50meters of all 
+    Exclusion Zones.
+    The responseCode = 400 for CBSDs located within 50meters of all Exclusion Zones 
+    or inside Exclusion Zones. 
+    """
+    config = loadConfig(config_filename)
+    # Very light checking of the config file.
+    self.assertNotEqual(len(config['exclusionZoneRecords']) , 0, "Exclusion zone records are zero !!!")
+
+    # Get exclusion zone records from config file.
+
+    for i in range(len(config['exclusionZoneRecords'])):
+       # Inject EXZ database records
+       exzData = {'exclusionZone': config['exclusionZoneRecords'][i]}
+       self._sas_admin.InjectExclusionZone({'record': exzData},config['frequencyRange'][i])    
+
+
+    # Register N2 devices
+    cbsd_ids_N2 = self.assertRegistered(config['registrationRequests_N2'])
+
+    # Register N3 devices
+    cbsd_ids_N3 = self.assertRegistered(config['registrationRequests_N3'])
+
+    # Register N4 devices
+    cbsd_ids_N4 = self.assertRegistered(config['registrationRequests_N4'])
+
+    # Execute CPAS and wait untill completion
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    # Generating grant requests for N2 and validating responses
+    grant_request = []
+    for i in range(len(config['grants_N2'])):
+        config['grants_N2'][i]['cbsdId'] = cbsd_ids_N2[i]
+        grant_request.append(config['grants_N2'][i])
+
+    request = {'grantRequest': grant_request}
+    response = self._sas.Grant(request)['grantResponse']
+    self.assertEqual(len(response), len(grant_request))
+
+    for response_num, resp in enumerate(response):
+          self.assertEqual(resp['cbsdId'], cbsd_ids_N2[response_num])
+          self.assertTrue('grantId' in resp)
+          #self.assertEqual(resp['response']['responseCode'], 400)
+
+
+    # Generating grant requests for N3 and validating responses
+    grant_request = []
+    for i in range(len(config['grants_N3'])):
+        config['grants_N3'][i]['cbsdId'] = cbsd_ids_N3[i]
+        grant_request.append(config['grants_N3'][i])
+
+  
+    request = {'grantRequest': grant_request}
+    response = self._sas.Grant(request)['grantResponse']
+    self.assertEqual(len(response), len(grant_request))
+
+    for response_num, resp in enumerate(response):
+          self.assertEqual(resp['cbsdId'], cbsd_ids_N3[response_num])
+          self.assertTrue('grantId' in resp)
+          #self.assertEqual(resp['response']['responseCode'], 400)
```

Comment:
Why is this commented out? (same question below)

Reply:
It was done for testing. It has been removed.